src/Bicep.Cli/Commands/RootCommand.cs

```diff
@@ -74,28 +75,31 @@ bicep build file.bicep --outdir dir1
       bicep build file.bicep --outfile file.json
       bicep build file.bicep --no-restore
       bicep build file.bicep --diagnostics-format sarif
+      bicep build --pattern './dir/*.bicep'
 
-  {exeName} format [options] <file>
+  {exeName} format [options] [<file>]
```

Comment:
would
{exeName} format [options] &lt;file> | --pattern &lt;pattern>
be better?  Technically it's part of options, but it guides a bit more.